resources/authgear/templates/en/web/authflowv2/__password_strength_meter.html

```diff
@@ -1,14 +1,16 @@
 {{ define "authflowv2/__password_strength_meter.html" }}
-<div class="{{ $.Classname }} password-strength-meter" data-password-policy-target="currentMeter">
-    <span class="password-strength-meter__icon material-icons mr-1"></span>
-		<span
-			class="password-strength-meter-description"
-			data-password-policy-target="currentMeterDescription"
-			data-desc-1="{{ template "v2-password-policy-password-strength-meter-1" }}"
-			data-desc-2="{{ template "v2-password-policy-password-strength-meter-2" }}"
-			data-desc-3="{{ template "v2-password-policy-password-strength-meter-3" }}"
-			data-desc-4="{{ template "v2-password-policy-password-strength-meter-4" }}"
-			data-desc-5="{{ template "v2-password-policy-password-strength-meter-5" }}"
-		></span>
-</div>
+<meter
```

Comment:
Let's define a new controller responsible for the look of this meter. Let's call it PasswordStrengthMeterController. The interface of this Controller is

- Assume `element` is HTMLMeterElement.
- Assume `value` is in range `[1,5]`
- It changes the appearance according to `value`.

And PasswordPolicyController is responsible for setting the `value` of the meter only.